lib/posix-socket/include/uk/socket_driver.h

```diff
@@ -620,7 +620,8 @@ posix_socket_family_count(void);
  * Creates a static struct posix_socket_driver for the AF family
  */
 #define _POSIX_SOCKET_FAMILY_REGISTER(lib, fam, vops)			\
-	__used __section("." _POSIX_SOCKET_FAMILY_SECNAME(lib, fam))	\
+	__used __align8							\
```

Comment:
Please also use `__align(8)` here (or replace the annotation at the struct with `__align8`).

```suggestion
	__used __align(8)						\
```